src/integration-test/java/de/danielbechler/diff/CategoriesTestIT.groovy

```diff
@@ -0,0 +1,133 @@
+package de.danielbechler.diff
```

Comment:
Let's move this to a new package `categories`, since all the other major configuration sections got their own one as well.